data-raw/watershed/clear/clear_creek.rmd

```diff
@@ -99,170 +83,90 @@ sr_spawn <- bind_rows(clear_raw_la, clear_raw_c, clear_raw_ua) %>%
+  labs(x = 'Flow (cfs)', y = 'WUA (sqft/1000ft)') + 
+  theme_minimal() + 
+  scale_color_manual(values = c('#d95f02','#7570b3', '#56B4E9')) +
+  theme(legend.justification = c(1,0), legend.position = c(.9,.55))
 
-```{r, echo=FALSE}
-clear_creek_instream %>% 
-  select(-FR_spawn_wua, -SR_spawn_wua, -ST_spawn_wua, -FR_fry_wua, -FR_juv_wua, -ST_fry_wua, -ST_juv_wua) %>%
-  gather(Species, wua, -flow_cfs) %>%
-  ggplot(aes(flow_cfs, wua, color = Species)) +
-  geom_line() +
-  labs(x = 'Flow (cfs)', y = 'WUA (sqft/1000ft)') +
-  scale_y_continuous(labels = comma) +
-  theme_minimal()+
-  scale_color_manual(values = c('#d95f02','#7570b3'))
 ```
 
-#### Steelhead Rearing WUA Plot
-
-The following plot shows the weighted usable rearing area in square feet per thousand feet of river for Steelhead fry and juvenile. These rates are multiplied by the total rearing reach length mapped by the SIT. Steelhead and Spring run juvenile rearing habitat are the same. The data is lumped together because Steelhead and Spring Run Salmon have the same criteria and there is a small sample size for both steelhead and spring run.
-
+### Rearing WUA
+The following plot shows the weighted usable rearing area in square feet per thousand feet of river for Fall Run Chinook Salmon, Spring Run Chinook Salmon, and Steelhead fry and juvenile. The juvenile rearing data is the same for Spring Run and Steelhead. These rates are multiplied by the total rearing reach length mapped by the SIT.
 ```{r, echo=FALSE}
 clear_creek_instream %>% 
-  select(-FR_spawn_wua, -SR_spawn_wua, -ST_spawn_wua, -FR_fry_wua, -FR_juv_wua, -SR_juv_wua, -SR_fry_wua) %>%
-  gather(Species, wua, -flow_cfs) %>%
-  ggplot(aes(flow_cfs, wua, color = Species)) +
+  select(FR_juv_wua, FR_fry_wua, ST_juv_wua, ST_fry_wua, SR_juv_wua, SR_fry_wua, 
+         flow_cfs, watershed) %>%
+  gather(species_lifestage, acres, -flow_cfs, -watershed) %>% 
+  mutate(
+    species = case_when( 
+      str_detect(species_lifestage, 'ST') ~ 'Steelhead', 
+      str_detect(species_lifestage, 'FR') ~ 'Fall Run Chinook',
+      str_detect(species_lifestage, 'SR') ~ 'Spring Run Chinook'),
+      lifestage = ifelse(str_detect(species_lifestage, 'juv'), 'Juvenile', 'Fry')) %>% 
+  ggplot(aes(flow_cfs, acres, color = species)) +
   geom_line() +
-  labs(x = 'Flow (cfs)', y = 'WUA (sqft/1000ft)') +
-  scale_y_continuous(labels = comma) +
-  theme_minimal()+
-  scale_color_manual(values = c('#d95f02','#7570b3'))
+  theme_minimal() +
+  facet_wrap(~lifestage, ncol = 1, scales = 'free_y') +
+  scale_color_brewer(palette = 'Dark2') +
+  labs(x = 'Flow (cfs)', y = 'Suitable Acres') + 
```

Comment:
the y axis label should be  'WUA (sqft/1000ft)', right?

Reply:
fixed